File: Examples/test-suite/objc/abstract_inherit_ok_runme.m
#import "abstract_inherit_ok_proxy.h"
#import <assert.h>
int main(int argc,char *argv[])
{
	NSAutoreleasePool *pool=[[NSAutoreleasePool alloc] init];
 //Shouldn't be able to instantiate Foo,because it declares a pure virtual function 
	
	Spam *sp=[[Spam alloc] init];
	assert([sp blah] == 0);
	NSLog(@"Spam Object method");
	[sp release];
	[pool drain];
	
	return 0;
}

File: Examples/test-suite/c/abstract_inherit_ok_runme.cxx
#include "abstract_inherit_ok_wrap.h"
#include <assert.h>

int main(int argc, const char *argv[]) {
    abstract_inherit_ok_Foo* const spam = (abstract_inherit_ok_Foo*)abstract_inherit_ok_Spam_new();

    assert(abstract_inherit_ok_Foo_blah(spam) == 0);

    abstract_inherit_ok_Foo_delete(spam);

    return 0;
}
